sirepo/feature_config.py

```diff
@@ -151,6 +156,9 @@ def b(msg, dev=False):
         if v[0] in s:
             s.add(v[1])
     _cfg.sim_types = frozenset(s)
+    if 'raydata' in _cfg.sim_types:
+        assert _cfg.raydata.data_dir, \
```

Comment:
I've been using https://github.com/radiasoft/raydata/blob/f46da6a9d89c6cc14be8c342051c8de9afa51bb1/etc/sirepo-dev.sh which follows our pattern of using scripts to start the server with interesting configuration. But it breaks the pattern by having the script in raydata not in sirepo. It could be moved into sirepo/etc. I think even with defaulting we are going to need a script because raydata needs to be put on sim_types and possibly added to package_path. So, my feeling is better to have less code in feature_config and push it out to the script.

Reply:
If we are working under the model that the system for the phase 1 really only works with one user then we will never get here (we will get the sate from the supervisor) and this is good enough. We really don't even need any locking in that case...

Reply:
Decided script was fine.